package.json

```diff
@@ -4,12 +4,10 @@
   "description": "script loader for module components",
   "main": "src/index.js",
   "scripts": {
```

Comment:
Can we add `files` to specify files for publish? Currently the configs and tests will be included in the published package. 
Doc: https://docs.npmjs.com/files/package.json#files
Example: https://github.com/vuejs/vue/blob/dev/package.json#L10